syntax/dirvish.vim

```diff
@@ -2,14 +2,13 @@ if 'dirvish' !=# get(b:, 'current_syntax', 'dirvish')
   finish
 endif
 
-let s:sep = exists('+shellslash') && !&shellslash ? '\' : '/'
-let s:sep_esc = s:sep == '\' ? '\\' : '/'
+let s:sep = escape(exists('+shellslash') && !&shellslash ? '\' : '/', '\\')
```

Comment:
@habamax @justinmk sorry, maybe I have tested wrong branch in Windows (not the OS I usually use), so I don't find the error earlier.

This line may be changed to `let s:sep = exists('+shellslash') && !&shellslash ? '\\' : '/'`? (no need to call `escape`)

Reply:
indeed, let me fix it